dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss

```diff
@@ -0,0 +1,3913 @@
+/**
+	Variables
+*/
```

Comment:
I guess that indentation changed and git treat this as new file. 
@sudmar Will you be able to merge your changes regarding font size? (you probably have quite big changes in scss) (and set your ide to use spaces instead of tabs for indentation)

Reply:
It was a file rename, it should not have changed the file contents. The file in `master` also has tabs: https://github.com/CeON/dataverse/blob/develop/dataverse-webapp/src/main/webapp/resources/vecler/theme.scss
Not sure though why it is not presented here as a rename, but as a new file.

Reply:
Oh. You're right.
Reason why it is not detected as file rename is probably because we have rename + theme.scss file create.
But I guess then merge would be easier then